packages/calypso-products/src/constants/titan.ts

```diff
@@ -1 +1,2 @@
 export const TITAN_MAIL_MONTHLY_SLUG = 'wp_titan_mail_monthly';
+export const TITAN_MAIL_ANNUALLY_SLUG = 'wp_titan_mail_annually';
```

Comment:
This needs to be `wp_titan_mail_yearly` - we probably want to use `YEARLY` in the constant as well.